build_tools/azure/conda/linux.yml

```diff
@@ -0,0 +1,30 @@
+steps:
```

Comment:
**Nit:** I see a lot of these steps being candidates for a template so that it can be abstracted and shared across multiple OS-specific yamls.

Reply:
I agree in principle, but in practice, it was a little harder. Each OS has its own quirks:

1. macOS requires `sudo` for setting up conda, but otherwise is the same as Linux
2. Linux and macOS have to build a wheel _before_ building conda
3. Windows requires a little bit of powershell, and to build conda from source.

That is the [way I initially had it](https://dev.azure.com/tgsmith61591gh/pmdarima/_build/results?buildId=1190), but I didn't really like how there were 2 or 3 OS-specific lines for some steps. Happy to change it if y'all prefer it that way!